server/db/models/comparison.js

```diff
@@ -0,0 +1,27 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Comparison = db.define('comparison', {
+  home: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  work: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  radius: {
+    type: Sequelize.INTEGER
```

Comment:
we should set a default radius in the future so that the map can render something